cassandra-scheduler/src/test/resources/scheduler.yml

```diff
@@ -1,17 +1,19 @@
 framework_name : ${FRAMEWORK_NAME:-"cassandra"}
 framework_version : ${FRAMEWORK_VERSION:-"version"}
 executor:
-  command : ${EXECUTOR_COMMAND:-"./executor/bin/cassandra-executor server executor/conf/executor.yml"}
+  command : ${EXECUTOR_COMMAND:-"ulimit -l $CASSANDRA_ULIMIT_MEMLOCK && ulimit -n $CASSANDRA_ULIMIT_NOFILE && ulimit -u $CASSANDRA_ULIMIT_NPROC && ulimit -a && ./executor/bin/cassandra-executor server executor/conf/executor.yml"}
```

Comment:
@verma7:  You're absolutely right.  Long story short, I leaked code related to a different approach to setting ulimits.  Almost everything should be removed.  Did you want to submit a PR?